bundles/org.openhab.binding.wled/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -41,37 +28,77 @@
 				<description>Use this format http://192.168.1.2:80</description>
 			</parameter>
 			<parameter name="pollTime" type="integer" required="true" min="1" unit="s">
```

Comment:
When you provide a default value, this is not necessary to mark the parameter as required.
Same remark for all the parameters.